.travis.yml

```diff
@@ -21,4 +24,5 @@ matrix:
           packages:
             - devscripts
             - debhelper
+            - moreutils
```

Comment:
Yes, it turns out this isn't on any of the targets we test on. See https://travis-ci.org/github/backup-utils/builds/206119031 for build failures before I added this to the travis hosts.